cmd/clusterctl/cmd/generate_provider.go

```diff
@@ -104,7 +104,7 @@ func runGenerateProviderComponents() error {
 
 	options := client.ComponentsOptions{
 		TargetNamespace:     gpo.targetNamespace,
-		SkipTemplateProcess: gpo.raw,
+		SkipTemplateProcess: gpo.raw || gpo.textOutput,
```

Comment:
Interesting mapping from describe => textOutput :joy: (flag name => var)

Reply:
Yes, I thought that myself but I wasn't sure if a rename was in scope. I can add the rename if you think it's worth it.

Reply:
Not sure why it was named like this originally, maybe there's a reason that we don't see.

Let's see if someone else comments (@fabriziopandini @ykakarap )

Reply:
This command predates me and I am not aware of the original reasoning behind that naming. From the git history of the file I couldn't find any historic reasons for naming it that way. My guess would be that the variable is named `textOutput` to match the associated function named `printComponentsAsText`.

Reply:
Should we resolve this and, if we think a rename is needed, open an issue or do it in another PR?

Reply:
Just resolving it is absolutely fine